.env.example

```
@@ -28,3 +28,6 @@ NEXT_PUBLIC_POSTHOG_HOST=
 NEXT_PUBLIC_ILLUSTRATION_GROUP_LINK=
 NEXT_PUBLIC_GASTRONOMY_GROUP_LINK=
 NEXT_PUBLIC_ENTREPRENEURSHIP_GROUP_LINK=
+
+# Secret for cron job authorization
+CRON_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)